openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -61,7 +61,8 @@ RUN if [ -z "${RELEASE_TAG}" ]; then \
     fi && \
     wget https://github.com/${RELEASE_ORG}/openvscode-server/releases/download/${RELEASE_TAG}/${RELEASE_TAG}-linux-${arch}.tar.gz && \
     tar -xzf ${RELEASE_TAG}-linux-${arch}.tar.gz && \
-    mv -f ${RELEASE_TAG}-linux-${arch} ${OPENVSCODE_SERVER_ROOT} && \
+    if [ -d "${OPENVSCODE_SERVER_ROOT}" ]; then rm -rf "${OPENVSCODE_SERVER_ROOT}"; fi && \
+    mv ${RELEASE_TAG}-linux-${arch} ${OPENVSCODE_SERVER_ROOT} && \
```

Comment:
Is this o3-mini related?

Reply:
This is not, but it fixes an issue in main when running evals. We could keep them together in this PR or I can merge this on a separate PR

Reply:
Cool, I’m just a passerby who recently tried to add `o3-mini` support to this project. While working on it, I came across this PR and thought I’d check it out. Keeping a clean Git history is usually helpful for troubleshooting and makes it easier for others to jump in, so I just wanted to share that thought. Of course, it’s totally up to you as the Collaborator. Thanks for reading!